modules/waf/default.auto.tfvars

```diff
@@ -0,0 +1,7 @@
+# This file is included by default in terraform plans
+
+enabled = false
+
+name = "waf"
```

Comment:
please move `name` and `default_action` from here to YAML stack config in the catalog for the component

Reply:
Updated Readme